tools/gulptasks/dashboards/scripts.js

```diff
@@ -70,7 +70,7 @@ async function dashboardsScripts() {
                 ),
                 namespace: 'Dashboards',
                 output: bundleTargetFolder,
-                version: (argv.release || '0.9.9')
```

Comment:
Which version should be the default? Should we update it each time?

Reply:
I've added this only to test the designs with the proper icons. 
Finally, it is going to be replaced in the `highcharts-utils` we have dedicated issue in the Assana for that.